src/LegacyPortal.js

```diff
@@ -0,0 +1,56 @@
+// This file is a fallback for a consumer who is not yet on React 16
+// as createPortal was introduced in React 16
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import { canUseDOM } from './utils';
```

Comment:
This is not used?

Reply:
ya, seems like it now that `renderPortal` is not being called in render